src/tool.h

```diff
@@ -47,6 +48,9 @@ struct ToolGroupCap
 		*time = i->second;
 		return true;
 	}
+
+	Json::Value toJson() const;
```

Comment:
don't return Json::Value instead use a Json::Value pointer as a param and make this function void